File: doc/codes/csharp/examples/api/compute/pattern/null-example.cs
using System.Numerics;
using AUTD3;

// HIDE
namespace DocSamples.ApiComputePatternNullExample;

internal static class Sample
{
    internal static void Run()
    {
        // HIDE_END
var geometry = new Geometry(new[] { new Autd3(Vector3.Zero) });

var dst = geometry.PatternBuffer();

Pattern.Null(dst);
        // HIDE
    }
}
// HIDE_END

File: doc/codes/python/examples/api/compute/pattern/null-example.py
from autd3.geometry import Autd3, Geometry
from autd3_pattern import null

geometry = Geometry([Autd3([0.0, 0.0, 0.0], [1.0, 0.0, 0.0, 0.0])])

out = geometry.pattern_buffer()

null(out)
